subprojects/packagefiles/expat/meson.build

```diff
@@ -153,23 +130,16 @@ _files = [
   ),
 ]
 
-# the lib/libexpat.def file is not valid with mingw, only with mvsc, intel-cl,
-# and clang-cl
-extra_kwargs = {}
-if cc.get_id() != 'gcc'
-  extra_kwargs += {'vs_module_defs' : 'lib/libexpat.def'}
-endif
-
 libexpat = library(
   'expat',
   _files,
   include_directories : include_directories('lib'),
   dependencies : dep_libbsd,
-  version : '1.6.11',
+  version : '1.8.8',
```

Comment:
Better define that at the top of the file, so we are less likely to forget when bumping version in project().

Reply:
Better solution would be to derive it from the project version. Not too sure how to do so. @eli-schwartz knows more I think.

Reply:
The library version is libtool `9:8:8`, which is mathematically expressed as 1.8.8, it has no basis in the project version.

Upstream just uses code comments to remember to bump it in every file whenever editing it in one.

Reply:
This was fixed.